packages/components/src/Spinner/Spinner.mdx

```diff
@@ -0,0 +1,54 @@
+---
+name: Spinner
+menu: Components
+route: /components/spinner
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Spinner } from ".";
+
+# Spinner
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Spinners are used to visualize an undetermined load time for a section of a page
+and prevent double clicking of a UI element by overlaying it while its action is
+processing.
```

Comment:
Some more detail to flesh out these docs... 

- The default spinner should be used to indicate loading of a view
- The `small` spinner should be used on individual elements of the interface (ie a calendar "pill", a button, a card's contents) or when inline with content (ie a message that says "Uploading Clients... {spinner}").